dockertest/subtest.py

```diff
@@ -24,6 +24,11 @@
 import version
 import config
 import subtestbase
+# There is a bug in Pylint + virtualenv that makes this fail in Travis CI
+# https://github.com/PyCQA/pylint/issues/73
+from distutils.version import LooseVersion  # pylint: disable=E0611,F0401
```

Comment:
Could save some future grief by just using the one from the already imported ``version`` module.  At some point (I hope) pylint will be fixed in travis (and match the versions we use in RHEL).  It's a minor issue though.